src/main/resources/static/api.yaml

```diff
@@ -723,4 +765,64 @@ components:
         telephone:
           type: string
         type:
-          type: string
\ No newline at end of file
+          type: string
+    ConsentRequest:
+      type: object
+      properties:
+        consent:
+          type: object
+          properties:
+            purpose:
+              type: object
+              properties:
+                text:
+                  type: string
```

Comment:
maybe add description that this is one from a fixed set, documented at the refUri